pokedex/src/Components/Pokeinfo/Pokeinfo.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "../Pokeinfo/Pokeinfo.css";
+import { v4 as uuidv4 } from "uuid";
+
+const Pokeinfo = ({ data }) => {
+  console.log(data);
+  return (
+    <>
+      {!data ? (
+        ""
+      ) : (
+        <>
+          <h1>{data.name}</h1>
+          <img
+            src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/dream-world/${data.id}.svg`}
+            alt=""
+          />
+          <div className="abilities">
+            {data.abilities.map((poke) => {
+              const id = uuidv4();
```

Comment:
`const id = uuidv4();` should not be used while rendering elements inside `jsx` because it changes the `key` every time the component rerenders

Please, use something fixed and unique